fmc/README.md

```diff
@@ -239,6 +239,23 @@ The following list of steps are to be performed by FMC on each boot when ROM jum
 
 <center>
 
+**Pre-Conditions:**
```

Comment:
Can you also add the same table showing the state AFTER FMC runs?

Reply:
Done